### ISSUE TITLE
Codegen for 2.5.0.44247

### ISSUE THREAD

**Issue by JackCeparou** [Enigma32/Enigma.D3#4]
Hi,

The code generator is crashing on a null reference inside Enigma.D3.CodeGen/Memory/Generator.cs
```csharp
globals.Add("const int Offset_PlayerData_HeroName", GetOffset_PlayerData_HeroName(sizeof_playerdata).ToHex());
```

Also it can't find the AttributeDescriptors :
```csharp
#error Could not find AttributeDescriptors :(
public const int AttributeDescriptors = 0x00000000;
```
Thanks for this wonderful tool 🥇

**Comment by Enigma32** [comment]
It will take a day or two. Having some issues with my virtual machine for IDA work, can't get Windows to start properly :(

**Comment by Enigma32** [comment]
Fixed with 8cdc80cb22d9e92905701b3d4248ba20c2287d2c
